app/templates/index/mygoals.hbs

```diff
@@ -0,0 +1,10 @@
+{{#fbf-text title=true}}
+  Your Goals
+{{/fbf-text}}
+{{#each model as |goaloverview|}}
```

Comment:
I would rename this template to something like `goal-list.hbs` and specify that it should be used for rendering in the routes of mygoals, trending and new and pass the title in via the route (since the url maps to the title). That way, you don't need `newgoals.hbs` and `trending.hbs`.